deploy/determined_deploy/aws/templates/efs.yaml

```diff
@@ -4,35 +4,35 @@ Mappings:
   RegionMap:
     ap-northeast-1:
       Master: ami-0349224eed420a3b5
-      Agent: ami-0172f5479691feb3d
+      Agent: ami-04941344de22297c2
```

Comment:
I'm not super familiar with how we do this now. Did you go through and manually change all of these AMIs? If so that sounds like a bug in the making where we will make as mistake and break our product in a region that we don't have testing for.